src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -23,6 +23,8 @@ import type { Annotation, AnnotationPermission, FeedItemStatus } from '../../../
 import type { GetAvatarUrlCallback, GetProfileUrlCallback } from '../../../common/flowTypes';
 import type { SelectorItems, User } from '../../../../common/types/core';
 
+import IconAnnotation from '../../../../icons/two-toned/IconTaskGeneral';
```

Comment:
This is not the correct Icon, designers started the work on adding the right one to the assets and refreshing it along the way. To parallelize the process, I used the one from `Task` to get the review on code changes sooner.